README.md

```diff
@@ -167,6 +185,23 @@ spec:
   replicas: 3
 ```
 
+#### Custom Labels
+Adds values for labels:
+
+```yaml
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  annotations:
+    infinispan.org/podTargetLabels: svc-label1, svc-label2
```

Comment:
I'm not sure do we need to say, that changes annotations and labels in the CR can't be reflected now.
We need to implement part of the #517 for Services and Routes

Reply:
Hmmm... This sounds like something the user should know. If you modify labels in Infinispan CR those changes do not get propagated? Does that include deleting labels or only modifying an existing label?